lib/narn-lib.js

```diff
@@ -41,6 +59,9 @@ exports.getNpmArgs = narnArgs => {
       const saveType = yarnArgs.dev || yarnArgs.D ? "--save-dev" : "--save";
       npmArgs = [saveType, ...packages];
       break;
+    case "rm":
```

Comment:
`yarn rm` is not a valid yarn command, and the API for `narn` is intended to be exactly the same as the `yarn` syntax. Same goes for `r`, `un`, `up`, and `ls`. Adding these would introduce narn commands that work for projects that use npm/pnpm but not for those that use yarn.

I'm a bit hesitant to deviate from that because the main ergonomics provided by narn to developers is "Here's a thing you already know (yarn) - you can use it no matter what project you are in"

My preference is to keep the API to narn itself the same as the yarn CLI.

```suggestion
```

Reply:
I understand your point, It's fair, I was just trying to add support for all the npm/pnpm native commands :ok_hand:

Reply:
the update / upgrade command is another thing tho, i think we should support that alias, since it is too similar